### PR TITLE
Fix Misspellings Across Repository

### PR DESCRIPTION
**Pull request by elisoncrum**
## Description

This pull request does not modify any logic or introduce any code. Modifications across this repository are automated from my GNU Aspell script, I just like looking through code.

## Related Issue
No related open issue.

## Type of change
- [x] Spelling.

## Checklist before merging
N/A

### CODE REVIEW

**Comment by jeff-mccoy** [comment]
@elisoncrum thanks for the PR! Our process requires a core team member to first do a git sign-off with our key before another team member can approve the changes. You'll still keep authorship of the PR, but every commit in Zarf is signed by a core team member (requirement of our stakeholders).  Once another member reviews and the tests pass, the PR can be merged.

**Review by YrrepNoj — APPROVED** [review]
I could have sworn I had a spellchecker installed... Thanks for catching all my mistakes!

**Pull request merged**
